core/src/main/java/com/redhat/runtimes/inventory/auth/rbac/RbacRestClientRequestFilter.java

```diff
@@ -0,0 +1,39 @@
+/* Copyright (C) Red Hat 2023 */
+package com.redhat.runtimes.inventory.auth.rbac;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import jakarta.ws.rs.client.ClientRequestContext;
+import jakarta.ws.rs.client.ClientRequestFilter;
+import java.io.IOException;
+import java.net.URI;
+import java.util.Base64;
+
+// Derived from notifications-backend
+
+/**
+ * Filter to optionally add data on outgoing requests to the RBAC service. This is meant for local
+ * development and not production.
+ */
```

Comment:
Are we sure that we want this leaking into code that's going to get deployed to prod? Are there any credible alternatives?

Reply:
It doesn't serve a purpose for us currently, so I'll just remove it.